app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/SimpleSearchActivity.java

```diff
@@ -31,9 +43,63 @@
      *
      * @param view SimpleSearchActvity's view
      */
-    @OnClick(R.id.search)
-    void search(View view) {
+    @OnClick(R.id.search) void search(View view) {
+        User user = User.getUser(this);
+
         String firstName = mFirstNameEditText.getText().toString();
         String lastName = mLastNameEditText.getText().toString();
+
+        DBAPICaller dbapiCaller = new DBAPICaller(user, this);
+
+        List<String> query = new ArrayList<>();
+        query.add(firstName);
+        query.add(lastName);
+        query.add("");
+        query.add("");
+
+        dbapiCaller.simpleSearch(query);
+    }
+
+    /**
+     * Bundle people and move to SearchResults Activity if search successful
+     *
+     * @param people List of person models
+     */
+    @Override public void onSearchSuccess(List<Person> people) {
+        Intent intent = new Intent(this, SearchResultsActivity.class);
+        Bundle bundle = new Bundle();
+
+        bundle.putParcelable(BUNDLE_KEY, new SimpleResult(people));
+        intent.putExtras(bundle);
+        startActivity(intent);
+    }
+
+    @Override public void authenticateUserCallSuccess(boolean success, Person person) {
```

Comment:
@as-f Is this intentionally blank? It might be worth adding a comment so people don't get confused, since there isn't any code here.